tests/verifier/lib/UDS.pm

```diff
@@ -0,0 +1,321 @@
+# Container for common data.
+package UDS;
+
+use strict;
+use warnings;
+
+
+# Own modules
+use TE::Common;
+use TE::Funclet;
+use TE::Utility;
+
+
+our $test_suite_uds = 
+    [
+        {
+            name => 'tc1',
+            note => '#1 - ping-pong w/o arguments (default is udp)',
+            pre_proc => \&te_def_pre_proc,
+            server_proc => \&te_def_server_proc,
+            server_arg => 'sr -i TARGET() ',
+            client_proc => \&te_def_client_proc,
+            client_arg => 'pp -i TARGET() ',
+            result_proc => \&te_def_result_proc,
+            result_arg => {
+                server => {
+                    success => ['Test end', 'interrupted by', 'exit', 'UDS type is SOCK_DGRAM'],
+                    failure => ['Segmentation fault', 'Assertion', 'ERROR']
+                },
+                client => {
+                    success => ['Test ended', 'Summary: Latency is', 'Warmup stage \(sending a few dummy messages\)...', 'UDS type is SOCK_DGRAM'],
+                    failure => ['Segmentation fault', 'Assertion', 'ERROR', 'server down']
+                },
+            },
+            post_proc => \&te_def_post_proc,
+        },
+        {
+            name => 'tc2',
+            note => '#2 - ping-pong option --tcp',
+            pre_proc => \&te_def_pre_proc,
+            server_proc => \&te_def_server_proc,
+            server_arg => 'sr -i TARGET() --tcp',
+            client_proc => \&te_def_client_proc,
+            client_arg => 'pp -i TARGET() --tcp',
+            result_proc => \&te_def_result_proc,
+            result_arg => {
+                server => {
+                    success => ['Test end', 'interrupted by', 'exit', 'UDS type is SOCK_STREAM'],
+                    failure => ['Segmentation fault', 'Assertion', 'ERROR']
+                },
+                client => {
+                    success => ['Test ended', 'Summary: Latency is', 'Warmup stage \(sending a few dummy messages\)...', 'UDS type is SOCK_STREAM'],
+                    failure =>  ['Segmentation fault', 'Assertion', 'ERROR', 'server down'],
+                }
+            },
+            post_proc => \&te_def_post_proc,
+        },
+        {
+            name => 'tc3',
+            note => '#3 - ping-pong option -b10 --tcp',
+            pre_proc => \&te_def_pre_proc,
+            server_proc => \&te_def_server_proc,
+            server_arg => 'sr -i TARGET() --tcp',
+            client_proc => \&te_def_client_proc,
+            client_arg => 'pp -i TARGET() --tcp -b10',
+            result_proc => \&te_def_result_proc,
+            result_arg => {
+                server => {
+                    success => ['Test end', 'interrupted by', 'exit', 'UDS type is SOCK_STREAM'],
+                    failure => ['Segmentation fault', 'Assertion', 'ERROR']
+                },
+                client => {
+                    success => ['Test ended', 'Summary: Latency of burst of 10 messages', 'UDS type is SOCK_STREAM'],
```

Comment:
"UDS type ..." looks as invalid according last master and https://github.com/Mellanox/sockperf/pull/193/files#diff-442cdda1ab1d725b607a7d8100d86dafbf8148ef15d49c3406a9a776d1ab821bR41